### ISSUE TITLE
I am not a web dev but would love to have it..

### ISSUE THREAD

**Issue by fisforfaheem** [rammcodes/Dopefolio#5]
Can u please give me some time to set it up..in start after that i would update it myself

**Comment by rammcodes** [comment]
Sure, let me know if you face any problems while setting up so I can help you out with that.

**Comment by fisforfaheem** [comment]
Please set me up a enviroment...so i could use it soemhow i dont udnerstand hwo toi use it...deply it.. etc...i know some basic html css but this is different bro pleass help

**Comment by fisforfaheem** [comment]
???

**Comment by truezerox** [comment]
Hello fisforfaheem. 

did you find the way to setting it up already otherwise i would like to help you our with it ? just contact me trough Discord its the fastest way. (my discord ID is on my github profile)

**Comment by fisforfaheem** [comment]
Yes I figured out how to use netlify and edit stuff, Thanks so much

**Comment by truezerox** [comment]
Nice to hear that it worked. I hope you that you have a nice day.
